.github/ISSUE_TEMPLATE/requesting-new-package.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Requesting New Package
+about: Request a new package be added to Fedora CoreOS
+title: 'New Package Request: <package name>'
+labels: ''
+assignees: ''
+
+---
+
+Please answer the following questions about the package you are requesting:
```

Comment:
```suggestion
Please try to answer the following questions about the package you are requesting:
```

It's definitely a lot of questions. Sometimes some of them aren't relevant, etc. Let's lighten the language a bit.